EJ/Submissions/P_6

```diff
@@ -0,0 +1,4 @@
+#(2-6) Famous Quote 2
+Famous_Person="John Quincy Adams"
+message="“If your actions inspire others to dream more, learn more, do more and become more, you are a leader.”"
+print(Famous_Person,message)
```

Comment:
Quotes usually have the form. Make this follow convention
```
Person: "quote"
```
Or 
```
"Quote"


				-- person
```

Reply:
I made edits inside a different pull request(I don't know how to edit them in one pull request)